site2/docs/admin-api-non-partitioned-topics.md

```diff
@@ -1,159 +1,5 @@
-|msgThroughputIn|Same as msgRateIn but in bytes per second instead of messages per second|
-|msgRateOut|The sum of all local and replication consumers’ dispatch rates in messages per second|
-|msgThroughputOut|Same as msgRateOut but in bytes per second instead of messages per second|
-|averageMsgSize|Average message size, in bytes, from this publisher within the last interval|
-|storageSize|The sum of the ledgers’ storage size for this topic|
-|publishers|The list of all local publishers into the topic. There can be anywhere from zero to thousands.|
-|producerId|Internal identifier for this producer on this topic|
-|producerName|Internal identifier for this producer, generated by the client library|
-|address|IP address and source port for the connection of this producer|
-|connectedSince|Timestamp this producer was created or last reconnected|
-|subscriptions|The list of all local subscriptions to the topic|
-|my-subscription|The name of this subscription (client defined)|
-|msgBacklog|The count of messages in backlog for this subscription|
-|msgBacklogNoDelayed|The count of messages in backlog without delayed messages for this subscription|
-|type|This subscription type|
-|msgRateExpired|The rate at which messages were discarded instead of dispatched from this subscription due to TTL|
-|consumers|The list of connected consumers for this subscription|
-|consumerName|Internal identifier for this consumer, generated by the client library|
-|availablePermits|The number of messages this consumer has space for in the client library’s listen queue. A value of 0 means the client library’s queue is full and receive() isn’t being called. A nonzero value means this consumer is ready to be dispatched messages.|
-|replication|This section gives the stats for cross-colo replication of this topic|
-|replicationBacklog|The outbound replication backlog in messages|
-|connected|Whether the outbound replicator is connected|
-|replicationDelayInSeconds|How long the oldest message has been waiting to be sent through the connection, if connected is true|
-|inboundConnection|The IP and port of the broker in the remote cluster’s publisher connection to this broker|
-|inboundConnectedSince|The TCP connection being used to publish messages to the remote cluster. If there are no local publishers connected, this connection is automatically closed after a minute.|
-
-#### pulsar-admin
-
-The stats for the topic and its connected producers and consumers can be fetched by using the
-[`stats`](reference-pulsar-admin.md#stats) command, specifying the topic by name:
-
-```shell
-$ pulsar-admin topics stats \
-  persistent://test-tenant/namespace/topic \
-  --get-precise-backlog
-```
-
-#### REST API
-
-{@inject: endpoint|GET|/admin/v2/persistent/:tenant/:namespace/:topic/stats|operation/getStats}
-
-#### Java
-
-```java
-admin.topics().getStats(persistentTopic, false /* is precise backlog */);
-```
+sidebar_label: Topics2
```

Comment:
```suggestion
sidebar_label: Topics 2
```

Reply:
I don't think topics 2 is a good sidebar title. Could you please help refine the title?

Reply:
@Huanli-Meng thank you very much for your comments.
The body content in the current md file will be merged into another md file, and then this md file will be empty or removed.
So it will not be displayed in the sidebar. You can regard the current metadata as placeholder.